spec/tom_queue/worker_supervisor_spec.rb

```diff
@@ -64,9 +64,9 @@
       supervisor.before_fork = -> { before_hook_message.set "executing before fork" }
 
       forked_supervisor.start
-
       expect(before_hook_message.wait).to eq "executing before fork"
-      forked_supervisor.term
+
+      forked_supervisor.kill # TODO: Why do we need a kill here to prevent orphan processes and nowhere else in the file?
```

Comment:
Something to chat through @joshado . Adding `.kill` was the only way to stop the process being an orphan 🤷🏻